src/classes/OPP_OpportunityContactRoles_TEST.cls

```diff
@@ -144,7 +150,7 @@ private class OPP_OpportunityContactRoles_TEST {
         Contact con = UTIL_UnitTestData_TEST.getContact();
         insert con;
         
-        Contact[] createdContacts = [select AccountId from Contact where Id = :con.id];
+        Contact[] createdContacts = [SELECT AccountId, Account.npe01__SYSTEMIsIndividual__c FROM Contact WHERE Id = :con.id];
```

Comment:
why the need to add acc.npe01__SYSTEMIsIndividual__c to the query?  you aren't testing it later.  did you mean to add an assert?

Reply:
It's not necessary -- it was on my radar. I have removed it from the query.

Reply:
not a big deal, but I still see it in the pull request.  did you forget to push your change?

Reply:
Good catch. I have removed the fields and pushed the code. Keep me posted. Thanks.